manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -89,31 +88,43 @@ Then the source of the add-on package will be checked out into the ``src`` folde
 
 .. note::
 
-    You can also add Products.PDBDebugMode and plone.reload to your development-tools:
+    You can also add some development-tools like ``Products.PDBDebugMode`` and ``plone.reload`` to your buildout.
+    For some you currently need a source checkout though.
 
     .. code-block:: ini
 
         custom-eggs +=
-            Products.PDBDebugMode [zodb]
+            collective.package
+            Products.PDBDebugMode
+            Products.PrintingMailHost
             plone.reload
 
+        test-eggs +=
+            collective.package [test]
+
+        auto-checkout +=
+            collective.package
+            Products.PDBDebugMode
+
         [sources]
-        Products.PDBDebugMode = git ${remotes:collective}/Products.PDBDebugMode.git pushurl=${remotes:collective_push}/Products.PDBDebugMode.git branch=python3
+        Products.PDBDebugMode = git ${remotes:collective}/Products.PDBDebugMode.git pushurl=${remotes:collective_push}/Products.PDBDebugMode.git branch=master
 
+        [versions]
+        Products.PrintingMailHost = 1.1.1
 
 .. code-block:: shell
 
     ./bin/buildout -c local.cfg
 
 Now everything is prepared to work on the migration of the package.
 
-However, it is a good idea to now try
+However, for small packages or packages that have few dependencies, it is a good idea to try starting your instance now.
 
```

Comment:
Can we remove/change the word "however" ?
It is not needed here, cause it adds no extra value and some people could read that as an insult and this is not good style in terms of tone.

Reply:
Done in 573b21c